core/model/src/main/java/org/eclipse/sw360/antenna/model/artifact/facts/ArtifactVcsInfo.java

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright (c) Bosch Software Innovations GmbH 2019.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.sw360.antenna.model.artifact.facts;
+
+import org.eclipse.sw360.antenna.model.artifact.ArtifactFact;
+
+import java.util.Objects;
+import java.util.Optional;
+
+public class ArtifactVcsInfo implements ArtifactFact<ArtifactVcsInfo> {
+    private final VcsInfo vcsInfo;
+
+    public static class VcsInfo {
```

Comment:
Could we simply directly use ORT's VcsInfo class here instead?

Reply:
If we use the ORT's VcsInfo class then the antenna-model has a direct dependency to ORT's model.